src/components/core/a11y/focus.ts

```diff
@@ -16,8 +16,11 @@ export const IS_FOCUSABLE_QUERY = [
 // Note: the use of this function for more complex scenarios (with many nested elements) may be expensive.
 export function getFocusableElements(
   elements: HTMLElement[],
-  filterFunc?: (el: HTMLElement) => boolean,
-  findFirstFocusable?: boolean,
+  properties?: {
```

Comment:
the whole refactoring would not be needed anymore?

Reply:
True but I kinda like it more like this anyway, what do you think?

Reply:
me too, but it's just an option (includeInvisibleElements?: boolean;) we never use, so kind of unnecessary code.